server/test/routes/index.test.ts

```diff
@@ -1,3 +1,4 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
```

Comment:
Should we combine these two lines by writing `2022 - 2023`?

Reply:
My suggestion for how to handle the copyright year on our files was to just include the year of the original creation of that file, 2022 on files created in 2022, 2023 on files created in 2023, and so on. Using a range is equally as valid, but we should be consistent. Considering that this project will eventually have other people developing it, keeping consistency is probably easier with just having the year of file creation.

Reply:
line removed